src/environmentd/src/http/static/js/memory.jsx

```diff
@@ -25,22 +25,26 @@ async function query(sql) {
   return data;
 }
 
+function formatNameForQuery(name) {
+  return `'${name.replace('\'', '\'\'')}'`;
+}
+
 const { useState, useEffect } = React;
 
-function ReplicaView() {
+function ClusterView() {
   const [currentClusterName, setCurrentClusterName] = useState(null);
   const [currentReplicaName, setCurrentReplicaName] = useState(null);
   const [sqlResponse, setSqlResponse] = useState(null);
   const [loading, setLoading] = useState(true);
   const [error, setError] = useState(false);
 
   const queryClusterReplicas = `
-    SELECT
-    clusters.name AS cluster_name, replicas.name AS replica_name
+    SELECT DISTINCT ON(clusters.name)
```

Comment:
If there's more than 1 replica, this will choose a random one. We should either do what the previous PR did (list all replicas and let the user choose) or remove replicas completely from this PR (this is my guess for what we actually need here).

Reply:
We need to choose an arbitrary replica to use in `SET cluster_replica =...`. If you only run `SET cluster = ...`, without specifying a replica, mz returns an error 
```
ERROR:  log source reads must target a replica
HINT:  Use `SET cluster_replica = <replica-name>` to target a specific replica in the active cluster. Note that subsequent `SELECT` queries will only be answered by the selected replica, which might reduce availability. To undo the replica selection, use `RESET cluster_replica`.

```

Reply:
Ohh ok. Looks like [some](https://github.com/MaterializeInc/materialize/blob/bf49edba062879dc874cbc8411f90007216fe1a2/src/adapter/src/coord/sequencer.rs#L1934) queries require specifying an exact replica, and these queries trigger that. I'm in favor of showing all replicas then, because the comment there suggests it matters.